doc/classes/Array.xml

```diff
@@ -510,14 +510,14 @@
 			<argument index="1" name="accum" type="Variant" default="null">
 			</argument>
 			<description>
-				Calls the provided [Callable] for each element in array and accumulates the result in [code]accum[/code]. The method for [Callable] takse two arguments: current value of [code]accum[/code] and the current array element. If [code]accum[/code] is [code]null[/code] (default value), the method will use first element from the array as initial value.
+				Calls the provided [Callable] for each element in array and accumulates the result in [code]accum[/code]. The method for [Callable] takes two arguments: current value of [code]accum[/code] and the current array element. If [code]accum[/code] is [code]null[/code] (default value), the method will use the first element from the array as initial value.
```

Comment:
```suggestion
				Calls the provided [Callable] for each element in array and accumulates the result in [code]accum[/code].
				The callable's method takes two arguments: the current value of [code]accum[/code] and the current array element. If [code]accum[/code] is [code]null[/code] (default value), the method will use the first element from the array as initial value.
```
There's an ambiguity left: if `accum` is not null, is the first element skipped, or does it mean there's an extra iteration? (`accum + 1 + 2 + 3` or `accum + 2 + 3`?)